src/api/app/views/webui/projects/label_templates/index.html.haml

```diff
@@ -4,6 +4,8 @@
 
   .card-body
     %h3= @pagetitle
+    .mb-3
+      = render partial: 'webui/shared/label_global_list', locals: { project: @project }
```

Comment:
@eduardoj I would render this in the project overview, like we do for packages 
![image](https://github.com/user-attachments/assets/2072c39d-e237-4437-bc71-c3745e7d6db3)

Mixing this into the label template tab where the project wide templates are managed is confusing IMO

Reply:
![image](https://github.com/user-attachments/assets/975fe69b-d6a2-4942-acdc-35f135cec73f)